insights/specs/default.py

```diff
@@ -176,6 +176,7 @@ class DefaultSpecs(Specs):
     dmsetup_status = simple_command("/usr/sbin/dmsetup status")
     dnf_conf = simple_file("/etc/dnf/dnf.conf")
     dnf_modules = glob_file("/etc/dnf/modules.d/*.module")
+    dnf_module_list = simple_command("/usr/bin/dnf module list")
```

Comment:
Please check the comment regarding this command on the MR to the core-assets repo. Thanks.

Reply:
And I also forget this point, that it's required to add a corresponding item for this `dnf_module_list` to the `insights_archive.py` file as well (to support the old version insights client)

You can use the `mangle` command provided by the insights-core to get the simple_file name of this spec. e.g.
```
# mangle "/usr/bin/dnf module list"
insights_commands/dnf_module_list
```

Thanks.

Reply:
updated default.py to `dnf_module_list = simple_command("/usr/bin/dnf -C --noplugins module list")` and add `dnf_module_list = simple_file("insights_commands/dnf_-C_--noplugins_module_list")` to insights_archive.py